src/CommonITILObject.php

```diff
@@ -2497,7 +2497,7 @@ public function prepareInputForAdd($input)
     private function checkFieldsConsistency(array $input): bool
     {
         if (
-            array_key_exists('date', $input) && !empty($input['date'])
+            array_key_exists('date', $input) && $input['date'] != 'NULL'
```

Comment:
I think empty value should still be handled.
```suggestion
            array_key_exists('date', $input) && !empty($input['date']) && $input['date'] != 'NULL'
```